pacwrapper_test.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWrapPAC(t *testing.T) {
```

Comment:
`t.Parallel()` at the front of all tests where practicable.

Reply:
I agree that's a useful thing, but I'm not going to start a few tests like that and leave the rest alone. That is something that should be considered across the whole code base I think, and is out of scope for this PR.

Reply:
nice one @anzdaddy - I did not know about this (agreed with @camh- should be part of a separate refactor).
I wonder if it's worth while using suites and setting this kind of thing in `SetupTest`?